example/factories.ts

```diff
@@ -60,7 +60,11 @@ export const fileStreamingEndpointsFactory = new EndpointsFactory({
         response.status(400).send(error.message);
         return;
       }
-      if ("filename" in output) {
+      if (
+        output &&
+        "filename" in output &&
+        typeof output.filename === "string"
+      ) {
```

Comment:
⚠️ these checks are now required due to FlatObject type and explicit possibility of the `output` to be null